build/docker/entrypoint.sh

```diff
@@ -43,10 +43,13 @@ initialize_log_dir() {
 }
 
 copy_provided_plugins() {
-  if [ -d "/opt/plugins" ]; then
-    echo "Copying user-provided plugins"
-    cp -nR /opt/plugins/* ${OPENFIRE_DIR}/plugins/
-  fi
+  for file in /opt/plugins/*.jar; do # Workaround for https://github.com/koalaman/shellcheck/wiki/SC2144 that doesn't involve redirecting terminal output
+    if [ -f $file ]; then
+      echo "Copying user-provided plugins"
+      cp -nR /opt/plugins/*.jar "${OPENFIRE_DIR}/plugins/"
```

Comment:
What is going on here with that '*.jar' when you are looping with $file variable?

Reply:
You're not wrong.
Let me give that another... bash :)